calibration/NUTNRB/CGINS-NUTNRB-00240__20180607.csv

```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-240,CC_cal_temp,20,
+240,CC_cal_temp,20.00, source_file NUTNR-B_ISUS_SN_240_Calibration_Files_2018-06-07.zip > ISUS240E.CAL
```

Comment:
This one acted all funky when I did updates per original SW comment (similar to what we saw w METBK updates): 
"This is the correct cal file and replaces CGINS-NUTNRB-00240__20180531 in GitHub.
File CGINS-NUTNRB-00240__20180531 in GitHub should be deleted."
ACTION by RGT - deleted file w/ date of 20180531 but it looks like I renamed the file and added source file info.